apps/pie-docs/src/assets/styles/components/_navToggle.scss

```
@@ -29,6 +29,15 @@
     }
 }
 
+html:has(.c-navToggle-input:checked) body {
+    /* when modal active */
+    touch-action: none;
+    -webkit-overflow-scrolling: none;
+    overflow: hidden;
+    /* Other browsers */
```

What does `other browsers` refer to here? I'd guess the -webkit property above is chrome and the property below is firefox/safari?

If that's the case, I'd prob mark this as:

```suggestion
    /* non webkit browsers */
    overscroll-behavior: none;
```